portal/messages/es.json

```
@@ -234,9 +234,16 @@
     "amount": "Cantidad",
     "drawer": {
       "approving": "Aprobando {symbol}",
-      "heading": "Revisar Stake",
-      "subheading": "Revise y confirme su stake.",
-      "stake-token": "Hacer stake de {symbol}"
+      "stake": {
+        "heading": "Revisar Stake",
+        "subheading": "Revise y confirme su stake."
+      },
+      "stake-token": "Hacer stake de {symbol}",
+      "unstake": {
+        "heading": "Estamos desbloqueando tus tokens",
+        "subheading": "La red está procesando tu desbloqueo. Podrás acceder a tus tokens una vez completado el paso."
```

If we apply the suggestion from https://github.com/hemilabs/ui-monorepo/pull/1490/files#r2337974551 - this would be...

```suggestion
        "subheading": "La red está procesando tu desbloqueo. Podrás acceder a tus tokens una vez completada la transacción."
```